filebeat/Jenkinsfile.yml

```
@@ -42,7 +42,7 @@ stages:
           - "macos12 && x86_64"
         when:                  ## Override the top-level when.
             comments:
-                - "/test filebeat for macos"
+                  - "/test filebeat for macos"
```

The other blocks are indented with 4 spaces, I did not understand the purpose of this change.
```suggestion
                - "/test filebeat for macos"
```